test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -58,11 +62,15 @@ def test_smdebug_cpu(training, ec2_connection, region, cpu_only, py3_only):
         hide=True,
     )
 
-    ec2_connection.run(
+    test_output = ec2_connection.run(
         f"docker exec --user root smdebug-cpu /bin/bash -c '{test_script} {framework}'",
-        hide=True,
+        hide=True, warn=True, timeout=5000
```

Comment:
nit: this timeout won't ever be reached in PR context, due to the global timeout of 4860 seconds. It's fine to keep it if you think it makes sense to have this as a high timeout for other instance types.